index.html

```diff
@@ -1544,22 +1544,24 @@
 
 <div class="practice" data-audiences="user-agents">
 <span class="practicelab" id="principle-owned-devices-disclose-surveillance">
-[=User agents=] must not help a device [=administrator=] surveil the people
-using the devices they administrate without those people's knowledge. [=User
-agents=] should not tell a device [=administrator=] about user behavior except
+[=User agents=] must not help an [=administrator=] surveil the people
+using the devices or software they administer without those people's knowledge.
+[=User agents=] should not tell an [=administrator=] about user behavior except
 when that disclosure is necessary to enforce reasonable constraints on use of
-the device.
+the device or software.
 </span>
 </div>
 <div class="note">
 See [[[#guardians]]] for more detail on how this principle applies to vulnerable people with [=guardians=].
 </div>
 
-Computing devices have <dfn data-lt="device owner">owners</dfn>, who have
-<dfn>administrator</dfn> access to the devices in order to install and
-configure the programs that run on them. As a program running on a device,
-a [=user agent=] generally can't tell whether the [=administrator=] who has
-installed and configured it was authorized by the device's actual owner.
+Computing devices have <dfn data-lt="administrator">administrators</dfn>, who
+have privileged access to the devices in order to install and configure the
+programs that run on them. The <dfn data-lt="device owner">owner</dfn> of a
+device often authorizes the [=administrators=], but there are also ways to get
+administrator access without the [=device owner=]'s knowledge. Some [=user
```

Comment:
Regarding "there are also ways to get administrator access without the device owner's knowledge", the paragraph above requires user agents to disclose when they are acting on behalf of an administrator and that would apply here as well. Unless the intention is to say "the device owner might not have configured the user agent, but the user account's organization administrator may have and the device owner doesn't know what accounts users might log into on their device".

I would focus this more on the "account administrators also have powers" angle rather than the "device owner might be unaware" angle.

Reply:
Yeah ... how's this? I use the [=device owner=] term elsewhere in this section, so it's hard to just delete the sentence defining it, although maybe we could rejigger that in a different way.